lib/migrate/Migrator.js

```diff
@@ -169,7 +190,7 @@ class Migrator {
     });
   }
 
-  down(config) {
+  down(name, config) {
```

Comment:
can't the name be part of config to keep this change backwards-compatible?

Reply:
done